addons/fetchmail/models/fetchmail.py

```diff
@@ -44,7 +44,7 @@ class FetchmailServer(models.Model):
     original = fields.Boolean('Keep Original', help="Whether a full original copy of each email should be kept for reference "
                                                     "and attached to each processed message. This will usually double the size of your message database.")
     date = fields.Datetime(string='Last Fetch Date', readonly=True)
-    user = fields.Char(string='Username', readonly=True, states={'draft': [('readonly', False)]})
+    user_id = fields.Char(string='Username', readonly=True, states={'draft': [('readonly', False)]}, oldname='user')
```

Comment:
This one is not a Many2one

Reply:
And what name should it have?

Reply:
For example, `user_login` or similar

Reply:
Yep indeed, shouldn't end with `_id`. Good catch Pedro! `username` is also an option.

Reply:
Done

Reply:
:+1:

Reply:
any news on this @tde-banana-odoo ?